app/assets/stylesheets/components/_what_to_expect.scss

```diff
@@ -0,0 +1,40 @@
+.what-to-expect-body {
+  .what-to-expect-intro {
```

Comment:
This nesting will compile to CSS as `.what-to-expect-body .what-to-expect-intro { … }`. That feels overly specific. Is there a case where `.what-to-expect-intro` might exist elsewhere on the site? Or, could this be simplified to:

```scss
.what-to-expect-body h3 {
  // styles go here…
}

.what-to-expect-intro {
  // styles go here
}
```

Reply:
I did this because .what-to-expect-intro by itself is not specific enough for margin-top, and would be overwritten by the usdsw css rule 
    .usa-width-three-fourths > :first-child { 
        margin-top: 0; 
    }

Reply:
Ah, gotcha!